websites/A/Amazon Music/presence.ts

```diff
@@ -99,31 +141,47 @@ presence.on("UpdateData", async () => {
             presence.timestampFromFormat(currentTime)
         ));
 
-      const data: PresenceData = {
-        details: title,
-        state: artist,
-        largeImageKey: "logo",
-        startTimestamp: timestamps[0],
-        smallImageKey: paused == true ? "pause" : "play",
-        smallImageText:
+        presenceData.details = title,
+        presenceData.state = artist,
+        presenceData.largeImageKey = "logo",
+        presenceData.startTimestamp = timestamps[0],
+        presenceData.smallImageKey = paused == true ? "pause" : "play",
+        presenceData.smallImageText =
           paused == true ? (await strings).pause : (await strings).play,
```

Comment:
isnt paused already a bool? you dont need the == true just do ?

Reply:
oh oops lmao my retard ass